packages/story-editor/src/components/floatingMenu/elements/shared/icon.js

```diff
@@ -31,11 +31,11 @@ import {
 import ToggleButton from './toggleButton';
 
 const IconButton = memo(
-  forwardRef(function IconButton({ Icon, title, ...rest }, ref) {
+  forwardRef(function IconButton({ Icon, ...rest }, ref) {
```

Comment:
we want the button to get the title so we can reference it on the `document.ActiveElement` in karma tests